packages/paste-core/components/badge/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import axe from '../../../../../.jest/axe-helper';
+import {Badge} from '../src';
+import {getVariantStyles} from '../src/utils';
+
+expect.extend(matchers);
+
+describe('Badge', () => {
+  describe('utils', () => {
+    // @TODO add "describe" block for "getComputedTokenNames"
```

Comment:
What's this about? A little unclear

Reply:
Ah yes. Previously I had a helper/utility function called `getComputedTokenNames` that was strongly inspired by the equivalent helper in the `Avatar` package. 
We have since decided that for now we will release a single size `Badge` so I removed this utility for now. I left it as a `@TODO` note originally because I had a feeling it would be out of scope for stage 1.